packages/mantine/src/components/table/TableDateRangePicker.tsx

```diff
@@ -45,15 +45,15 @@ export const TableDateRangePicker: FunctionComponent<TableDateRangePickerProps>
     };
 
     const formatDate = (date: Date) => dayjs(date).format('MMM DD, YYYY');
-    const formatedRange = `${formatDate(form.values.dateRange[0])} - ${formatDate(form.values.dateRange[1])}`;
+    const formattedRange = `${formatDate(form.values.dateRange[0])} - ${formatDate(form.values.dateRange[1])}`;
 
     return (
         <Grid.Col span="content" order={TableComponentsOrder.DateRangePicker} py="sm">
             <Group spacing="xs">
-                <Text span>{formatedRange}</Text>
+                <Text span>{formattedRange}</Text>
                 <Popover opened={opened} onChange={setOpened} withinPortal>
                     <Popover.Target>
-                        <Button variant="outline" color="gray" onClick={() => setOpened(true)} px="xs">
+                        <Button variant="outline" color="gray" onClick={() => setOpened((o) => !o)} px="xs">
```

Comment:
[optional] I stumbled upon https://mantine.dev/hooks/use-toggle/ recently, I guess we could use it here since we're toggling. I don't mind the current solution either.